tests/main/chattr/task.yaml

```diff
@@ -1,6 +1,6 @@
 summary: test chattr
 # ubuntu-core doesn't have go :-)
-systems: [-ubuntu-core-16-64, -ubuntu-core-16-arm-64, -ubuntu-core-16-arm-32]
+systems: [-ubuntu-core-16-64, -ubuntu-core-16-arm-64, -ubuntu-core-16-arm-32, -ubuntu-16.04-ppc64el]
```

Comment:
Ideally each test disabled on `ppc64el` would be accompanied by a small note explaining why this is done.